### PR TITLE
OLS-836: Add Data, Telemetry, Transcript, and Feedback Collection to …

### PR DESCRIPTION
**Pull request by rh-tokeefe**
…the About section

Version(s): Tech Preview cp to TP1

Issue:
https://issues.redhat.com/browse/OLS-836

Link to docs preview:
https://79379--ocpdocs-pr.netlify.app/openshift-lightspeed/latest/about/ols-about-openshift-lightspeed.html#ols-about-data-telemetry-transcript-and-feedback-collection_ols-about-data-use

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by thoraxe** [comment]
LGTM

File: about/ols-about-openshift-lightspeed.adoc [hunk 1]
```diff
@@ -6,12 +6,13 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-This section provides an overview of {ols-full} and discusses functional requirements.
+The following topics provide an overview of {ols-full} and discusses functional requirements.
```
Comment (syedriko) [inline about/ols-about-openshift-lightspeed.adoc:9]: discuss

File: modules/ols-about-data-telemetry-transcript-and-feedback-collection.adoc [hunk 2]
```diff
@@ -0,0 +1,12 @@
+// This module is used in the following assemblies:
+// configure/ols-using-openshift-lightspeed.adoc
```
Comment (syedriko) [inline modules/ols-about-data-telemetry-transcript-and-feedback-collection.adoc:2]: // about/ols-about-openshift-lightspeed.adoc ?

File: about/ols-about-openshift-lightspeed.adoc (hunk 1 above)
Comment (rh-tokeefe) [reply about/ols-about-openshift-lightspeed.adoc:9]: done

File: modules/ols-about-data-telemetry-transcript-and-feedback-collection.adoc (hunk 2 above)
Comment (rh-tokeefe) [reply modules/ols-about-data-telemetry-transcript-and-feedback-collection.adoc:2]: yes, @syedriko  thank you for noticing this. I copied the header from a different file and forgot to change this.

Comment (rh-tokeefe) [reply modules/ols-about-data-telemetry-transcript-and-feedback-collection.adoc:2]: done

**Review by gabriel-rh — APPROVED** [review]
LGTM

**Pull request merged**